infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/DockerRuntimeContext.java

```diff
@@ -88,7 +85,7 @@ public DockerEnvironment getDockerEnvironment() {
   @Override
   public URI getOutputChannel() throws InfrastructureException {
     try {
-      return UriBuilder.fromUri(websocketEndpointBase).path(OUTPUT_WEBSOCKET_ENDPOINT_BASE).build();
+      return URI.create(websocketOutputEndpoint);
     } catch (UriBuilderException | IllegalArgumentException ex) {
```

Comment:
Looks like catching of UriBuilderException is redundant. The same in Openshift infra